ome_zarr/reader.py

```diff
@@ -265,7 +265,9 @@ def __init__(self, node: Node) -> None:
         axes_values = {"t", "c", "z", "y", "x"}
         try:
             multiscales = self.lookup("multiscales", [])
-            version = multiscales[0].get("version", "0.1")  # should this be matched with Format.version?
+            version = multiscales[0].get(
+                "version", "0.1"
+            )  # should this be matched with Format.version?
```

Comment:
I took a bit of a look yesterday. One possibility would be for this whole block to move into the `Format` hierarchy.

Reply:
In general I agree, but for me the relation between the classes deriving from `Spec`, e.g. `Multiscales`, and the `Format` classes is not quite clear. I think we would need a longer zoom session to go through the library design to discuss this, so I would suggest to not tackle this in the current PR and leave this part as is for now.